int16   num1
string  name1